contracts/MerkleTree.sol

```diff
@@ -59,6 +59,12 @@ contract MerkleTree is Whitelist {
       uint256 _zeroValue,
```

Comment:
We can remove this parameter since the contract computes the `ZERO_VALUE` anyway.